documentation/gitbook/how-to-run/advanced-run-guides/quick-start-guide-conda.md

```diff
@@ -80,62 +75,45 @@ In this `conda` environment, commands with R and python will uses this environme
 
 ### Define environment variables
 
-First, you'll need to fill in some variables that are used by the model. This can be done in a script (an example is provided at the end of this page). For your first time, it's better to run each command individually to be sure it exits successfully.
+Since you'll be navigating frequently between the folder that contains your project code and the folder that contains the core flepiMoP model code, it's helpful to define shortcuts for these file paths. You can do this by creating environmental variables that you can then quickly call instead of writing out the whole file path.
 
-First, in `myparentfolder` populate the folder name variables for the paths to the flepimop code folder and the project folder:
+If you're on a **Mac** or Linux/Unix based operating system, define the FLEPI\_PATH and PROJECT\_PATH environmental variables to be your directory locations, for example
 
 ```bash
-export FLEPI_PATH=$(pwd)/flepiMoP
-export DATA_PATH=$(pwd)/flepimop_sample
+export FLEPI_PATH=/Users/YourName/Github/flepiMoP
+export PROJECT_PATH=/Users/YourName/Github/flepiMoP/examples/tutorials
 ```
 
-Go into the code directory (making sure it is up to date on your favorite branch) and do the installation required of the repository:
+or, if you have already navigated to your flepiMoP directory
 
 ```bash
-cd $FLEPI_PATH # move to the flepimop directory
-Rscript build/local_install.R # Install R packages
-pip install --no-deps -e flepimop/gempyor_pkg/ # Install Python package gempyor
+export FLEPI_PATH=$(pwd)
+export PROJECT_PATH=$(pwd)/examples/tutorials
 ```
 
-Each installation step may take a few minutes to run.
-
-{% hint style="info" %}
-Note: These installations take place in your conda environment and not the local operating system. They must be made once while in your environment and need not be done for every time you run a model, provided they have been installed once. You will need an active internet connection for installing the R packages (since some are hosted online), but not for other steps of running the model.
-{% endhint %}
-
-<details>
-
-<summary>Help! I have errors in installation</summary>
-
-If you get an error because no cran mirror is selected, just create in your home directory a `.Rprofile` file:
+You can check that the variables have been set by either typing `env` to see all defined environmental variables, or typing `echo $FLEPI_PATH` to see the value of `FLEPI_PATH`.
 
-{% code title="~/.Rprofile" lineNumbers="true" %}
-```r
-local({r <- getOption("repos")
-       r["CRAN"] <- "http://cran.r-project.org" 
-       options(repos=r)
-})
-```
-{% endcode %}
+If you're on a **Windows** machine
 
-Perhaps this should be added to the top of the local\_install.R script #todo
+<pre class="language-bash"><code class="lang-bash"><strong>set FLEPI_PATH=C:\Users\YourName\Github\flepiMoP
+</strong>set PROJECT_PATH=C:\Users\YourName\Github\flepiMoP\examples\tutorials
+</code></pre>
```

Comment:
again, what part of this is now in the install action, such that we don't want them to do it?